src/Property.cpp

```diff
@@ -196,14 +204,14 @@ void Property::setValue(const char *value, bool updateToMqtt)
     }
     strcpy(_value, value);
 
-    log_v("Property %s(%s) topic->'%s' payload->'%s:::%p' bufferValue->'%s:::%p' ", _name, _id, _topic, value, &value, _value, &_value);
-    if (!_retained)
-        return;
+    log_i("Property %s(%s) topic->'%s' payload->'%s:::%p' bufferValue->'%s:::%p' ", _name, _id, _topic, value, &value, _value, &_value);
 
     if (updateToMqtt)
         _client.publish(prefixedPropertyTopic(_parent.getParent().getWorkingBuffer(), "/set"), 1, true, _value);
     else
         _client.publish(_topic, 1, true, _value);
+
+    publishPrefixedTopics();
```

Comment:
After setting a new value of the property all infos were removed. Or better after setting a new value on a property my tool "MQTT Explorer" does not show the info fields any longer.
I'll re-check that with an other tool.

Reply:
I am using MQTT Explorer as well and never encountered the problem.
Are you subscribed to just all topics?
Otherwise, I feel like it is broker related?